pkg/jx/cmd/common.go

```diff
@@ -285,6 +289,11 @@ func (o *CommonOptions) TeamAndEnvironmentNames() (string, string, error) {
 	return kube.GetDevNamespace(kubeClient, currentNs)
 }
 
+func (o *CommonOptions) ParseImagePullSecrets() []string {
+	pullSecrets := strings.Fields(o.PullSecrets)
+	return pullSecrets
+}
+
```

Comment:
Should be GetImagePullSecrets() to be consistent with other method names

Reply:
Agreed, thanks

Reply:
Nice. Unortunately that's as far as I can provide a sensible review ;-)